modules/planning/common/planning_gflags.cc

```diff
@@ -492,3 +492,5 @@ DEFINE_bool(enable_parallel_open_space_smoother, false,
             "True to enable open space smoother parallel implementation.");
 DEFINE_bool(enable_derivative_check, false,
             "True to enable derivative check inside open space planner.");
+DEFINE_bool(enable_hand_derivative, false,
```

Comment:
@deidaraho Please consider moving those planner specific flags to conf in the future.

Reply:
they are in planning.conf now.